server/service/src/invoice_line/stock_in_line/generate.rs

```diff
@@ -20,7 +20,6 @@ pub struct StockLineInput {
     pub barcode_id: Option<String>,
     pub supplier_link_id: String,
     pub overwrite_stock_levels: bool,
-    pub campaign_id: Option<String>,
```

Comment:
Have removed `campaign_id` from input. All of the impacted files actually use `generate_batch`, which now uses `campaign_id` from `InvoiceLineRow`

Impacted Files: 
`server/service/src/invoice/customer_return/update/generate.rs`
`server/service/src/invoice_line/inbound_shipment_from_internal_order_lines/generate.rs`
`server/service/src/invoice_line/stock_in_line/insert/generate.rs`
`server/service/src/invoice_line/stock_in_line/update/generate.rs`